packages/create-block/lib/index.js

```diff
@@ -133,72 +134,59 @@ program
 					);
 
 					if ( ! variant && availableVariants.length > 1 ) {
-						const result = await inquirer.prompt( {
-							type: 'list',
-							name: 'variant',
+						variant = await select( {
 							message:
 								'The template variant to use for this block:',
-							choices: availableVariants,
+							choices: availableVariants.map( ( value ) => ( {
+								value,
+							} ) ),
 						} );
-						variant = result.variant;
 					}
 
 					const defaultValues = getDefaultValues(
-						pluginTemplate,
+						projectTemplate,
 						variant
 					);
 
-					const filterOptionsProvided = ( { name } ) =>
-						! Object.keys( optionsValues ).includes( name );
-					const blockPrompts = getPrompts(
-						pluginTemplate,
+					const blockAnswers = await runPrompts(
+						projectTemplate,
 						[
 							'slug',
 							'namespace',
 							'title',
 							'description',
 							'dashicon',
 							'category',
-							'textdomain',
-						],
-						variant
-					).filter( filterOptionsProvided );
-					const blockAnswers = await inquirer.prompt( blockPrompts );
-
-					const pluginAnswers = plugin
-						? await inquirer
-								.prompt( {
-									type: 'confirm',
-									name: 'configurePlugin',
-									message:
-										'Do you want to customize the WordPress plugin?',
-									default: false,
-								} )
-								.then( async ( { configurePlugin } ) => {
-									if ( ! configurePlugin ) {
-										return {};
-									}
+							! plugin && 'textdomain',
```

Comment:
In my testing, the `textdomain` prompt only makes sense when `--no-plugin` option is selected. Otherwise, it should be equal to the plugin slug name.